packages/itwinui-react/src/core/Checkbox/Checkbox.tsx

```diff
@@ -116,13 +130,17 @@ export const Checkbox = React.forwardRef((props, ref) => {
           [`iui-${status}`]: !!status,
           'iui-loading': isLoading,
         },
-        className,
+        wrapperClassName,
       )}
-      style={style}
+      {...restWrapperProps}
     >
       {checkbox}
       {label && (
-        <Box as='span' className='iui-checkbox-label'>
+        <Box
+          as='span'
+          className={(cx('iui-checkbox-label'), labelClassName)}
```

Comment:
there is a huge (yet subtle) typo in this code

```suggestion
          className={cx('iui-checkbox-label', labelClassName)}
```

this should fix your issue above with tests and you should be able to revert images
> Image changes were added. Visual React test in CI won't pass without them, but I don't think the image changes originated from my PR. Any advice?

this also makes me realize you are missing a unit test for this important functionality.

Reply:
Good eye! Fixed the typo. 

> this also makes me realize you are missing a unit test for this important functionality.

What sort of unit test should I ad?

Reply:
same as the one for wrapperProps. test the behavior you are adding.

Reply:
Added.

Reply:
now update images too

Reply:
Done!